tests/console/validate_ext4_fs.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate ext4 filesystem and system partitioning
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+
+    select_console "root-console";
+
+    # Validate that swap is created and /home is on seperate partition.
+
+    assert_script_run("lsblk -n | grep 'SWAP'", fail_message => "/swap partition was not created.");
+    assert_script_run("lsblk -n | grep 'home'", fail_message => "/home is not on separate partition.");
+
+    # Validate that the filesystem type is the expected one.
+
+    my $test_data  = get_test_suite_data();
+    my @partitions = @{$test_data->{partitions}};
+
+    foreach (@partitions) {
+
+        # Validate that the filesystem type is the expected one.
+
+        record_info("Check fs $_->{mnt_point}", "Verify the '$_->{mnt_point}' partition filesystem is '$_->{fs_type}'");
+        my $fstype = script_output("df -PT $_->{mnt_point} | grep -v \"Filesystem\" | awk '{print \$2}'");
+        assert_equals($_->{fs_type}, $fstype, "File system on '$_->{mnt_point}' partition does not correspond to the expected one.");
+
+        # Validate that the size of root and home partition is the expected one.
+
+        record_info("Check fs $_->{mnt_point}", "Verify the '$_->{mnt_point}' partition size is '$_->{size}'");
```

Comment:
same here, for `Check size $_->{mnt_point}` so both boxes would look different in openQA when the `record_info` is displayed.